pkgs/servers/web-apps/discourse/plugins/discourse-solved/default.nix

```diff
@@ -9,8 +9,8 @@ mkDiscoursePlugin {
   src = fetchFromGitHub {
     owner = "discourse";
     repo = "discourse-solved";
-    rev = "e82c6ae1ca38ccebb34669148f8de93a3028906e";
-    sha256 = "sha256-KwpBCk4fjkZlSzgtc0OXvQupPd+aPb9nONyyALpEZhg=";
+    rev = "4f0234f5be3aaa77db277e0f224cd9750d2713cd";
```

Comment:
`discourse-solved` is now broken (found with `discourseAllPlugins`). It works with the previous revision.

Not sure if we're misinterpreting the `.discourse-compatibility` files.

Reply:
The update script doesn't understand explicit version operators like in `< 3.5.0.beta3-dev` so we have to adapt it. I just spent some time debugging this problem.

The syntax changed some time ago: https://meta.discourse.org/t/pinning-plugin-and-theme-versions-for-older-discourse-installs-discourse-compatibility/272665

Reply:
Rolled back the discourse-solved update. If you know what needs to be changed in the updater, please make the change.